CHANGELOG.md

```diff
@@ -1,130 +1,176 @@
-# Changelog
+# Change log
 
-## 2017-02-11 - Release 2.3.0
+All notable changes to this project will be documented in this file.
+Each new release typically also includes the latest modulesync defaults.
+These should not impact the functionality of the module.
 
-This is the last release with Puppet3 support!
-- Bump dependencies for Puppet4 support
+## [v3.0.0](https://github.com/voxpupuli/puppet-kafka/tree/v3.0.0) (2017-02-13)
+[Full Changelog](https://github.com/voxpupuli/puppet-kafka/compare/v2.3.0...v3.0.0)
+
+**Implemented enhancements:**
+
+- Add custom config dir [\#130](https://github.com/voxpupuli/puppet-kafka/pull/130) ([antyale](https://github.com/antyale))
+
+## [v2.3.0](https://github.com/voxpupuli/puppet-kafka/tree/v2.3.0) (2017-02-11)
+[Full Changelog](https://github.com/voxpupuli/puppet-kafka/compare/v2.2.0...v2.3.0)
+
+**Closed issues:**
+
+- broker.id should also be set in $logs\_dir/meta.properties [\#117](https://github.com/voxpupuli/puppet-kafka/issues/117)
+
+**Merged pull requests:**
+
+- release 2.3.0 [\#132](https://github.com/voxpupuli/puppet-kafka/pull/132) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
For everything else sure, I just find it a bit weird that the release merge for the previous version is under the changelog of the current one.